prov/efa/docs/efa_rdm_protocol_v4.md

```diff
@@ -430,6 +431,15 @@ These two fields were introduced with the extra request "connid in header". They
 therefore an implemenation is not required to set them. (section 4.4 for more details) If an implementation
 does set the connid, the implementation needs to toggle on the CONNID_HDR flag in `flags` (table 1.4).
 
+`connid` and `padding` fields are followed by an optional `host_id` (8 bytes) which is the sender's host id.
+If `connid` and `padding` are not present, `host_id` will follow `extra_info`.
+
+Table: 2.3 A list of handshake packet flags
+
+| Bit Id | Value | Name | Meaning |
+|---|---|---|---|
+|  0     | 0x1    | `HOST_ID_HDR` | This packet has the optional sender host id header |
```

Comment:
This flag is specific to handshake packet, therefore its name should be `HANDSHAKE_HOST_ID_HDR`.

Reply:
Thank you. Addressed.